scripts/templates/generate_coverage_report

```diff
@@ -112,6 +112,7 @@ if [[ "$1" == "--codecov" ]]; then
       exit 1
    fi
 
+
```

Comment:
Extra whitespace, please remove.
Nit: after making a change and saving the commit, just run: `git diff HEAD~1` to inspect your change` and make sure doesn't have any unnecessary things :-)